docs/SUMMARY.md

```diff
@@ -4,24 +4,27 @@
 
 ## Features
 
-* [🔥🌶️ Bitcoin Native Swaps 🌶️🔥](product-features/bitcoin-swaps/README.md)
+* [🪙 Bitcoin Native Swap](product-features/bitcoin-swaps/README.md)
```

Comment:
I would put the chile to the BNS, because it is a feature we want to highlight, perhaps only the chile on both sides..